*** ./build/test/smoke-pdftex.luatex.tlg	Wed Apr  8 16:47:12 2026
--- ./build/test/smoke-pdftex.luatex.log	Wed Apr  8 16:47:12 2026
***************
*** 1 ****
--- 1,87 ----
+ This is a generated file for the l3build validation system.
+ Don't change this file in any respect.
+ Package microtype Info: Loading generic protrusion settings for font family
+ (microtype)             `TeXGyrePagellaMath' (encoding: TU).
+ (microtype)             For optimal results, create family-specific settings.
+ (microtype)             See the microtype manual for details.
+ (../mt-msa.cfg
+ File: mt-msa.cfg ....-..-.. v... microtype config. file: AMS symbols (a) (RS)
+ ) (../mt-msb.cfg
+ File: mt-msb.cfg ....-..-.. v... microtype config. file: AMS symbols (b) (RS)
+ )
+ Package microtype Info: Character `ﬆ' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `ﬅ' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `‥' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `‼' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁰' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁴' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁵' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁶' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁷' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁸' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁹' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₀' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₁' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₂' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₃' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₄' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₅' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₆' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₇' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₈' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₉' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₊' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `₋' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁺' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
+ Package microtype Info: Character `⁻' is missing
+ (microtype)             in font `TU/TeXGyrePagella(0)/b/n/10'.
+ (microtype)             Ignoring protrusion settings for this character.
  TRYMTEX-PDFTEX-OK
